### ISSUE TITLE
Add support for shared drives?

### ISSUE THREAD

**Issue by sagan** [moisesmcardona/GoogleDriveUploadTool#17]
Hi

Would you like add support for Google drive's "Shared drives" (old name "Team drives") feature? It will allow users to upload files to shared drives that they have access to.

**Comment by cgkings** [comment]
good idea

**Comment by ntenga** [comment]
yeah i would also like to be able to access the shared drives with this program and upload/download. sorry i can't help with it. Hope you can manage to implement it
